docs/_static/wharf-ci-schema.json

```
@@ -250,6 +250,27 @@
                                     "SECOND_ARG=12453125"
                                 ]
                             ]
+                        },
+                        "secretName": {
+                            "type": "string",
+                            "description": "If specified, the target Kubernetes secret name will be mounted to be available for secret arguments via the secretArgs setting. The secret must have the name `wharf-${WHARF_INSTANCE}-project-${WHARF_PROJECT_ID}-secretname-${secretName}`.",
+                            "default": null,
+                            "examples": [
+                                "myproject-system-tests-secret"
```

Nit: Keep consistent with `docker.md` example.

```suggestion
                                "mysecret"
```